pkg/virt-controller/services/template.go

```diff
@@ -533,6 +554,9 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 		},
 		Resources: resources,
 		Ports:     ports,
+		Env: []k8sv1.EnvVar{
+			{Name: LessPvcSpaceTolerationEnvName, Value: strconv.Itoa(lessPvcSpaceToleration)},
```

Comment:
Can we make it an argument instead of an env var?

Reply:
very good idea, done